src/main/java/com/SpringBoot/springbootRef/Registration/Register.java

```diff
@@ -0,0 +1,117 @@
+package com.SpringBoot.springbootRef.Registration;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.SequenceGenerator;
+
+import java.util.Objects;
+
+@Entity
+public class Register {
+    @jakarta.persistence.Id
+    @SequenceGenerator(
+            name="user_id_sequence",
+            sequenceName="user_id_sequence"
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "user_id_sequence"
+    )
+    public Integer Id;
+    public String username;
+    private String password;
+    public Integer phonenumber;
```

Comment:
The integer data type does not support the length of the phone number. Therefore, changes need to be made.